src/pages/TransactionDuplicate/ReviewFields.tsx

```diff
@@ -0,0 +1,78 @@
+import React, {useMemo} from 'react';
+import {View} from 'react-native';
+import InteractiveStepSubHeader from '@components/InteractiveStepSubHeader';
+import SelectionList from '@components/SelectionList';
+import RadioListItem from '@components/SelectionList/RadioListItem';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import variables from '@styles/variables';
+import CONST from '@src/CONST';
+
+type FieldItemType = {
+    text: string;
+    value: string | boolean;
+    keyForList: string;
+};
+
+type ReviewFieldsProps = {
+    stepNames: string[];
+    label: string;
+    options: Array<{text: string; value: string | boolean}>;
+    index: number;
+    onSelectRow: (item: FieldItemType) => void;
+};
+
+function ReviewFields({stepNames, label, options, index, onSelectRow}: ReviewFieldsProps) {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+    let falsyCount = 0;
```

Comment:
Should we define this variable outside of the component?

Reply:
this change causing the bug, so I will leave it as it is